libred/red.c

```diff
@@ -459,6 +477,132 @@ static int call_posix_spawnp(pid_t *restrict pid, const char *restrict file,
 }
 #endif
 
+
+#define redirect_hardcoded(path, replacement)                         \
+  patterns[0] = "^/usr/bin/" path "$";                                \
```

Comment:
Using PCRE for simple string matching feels really heavyweight, I'd just do the matching manually.